dependencies/gmp/include/gmp.h

```diff
@@ -1,2329 +1,13 @@
-/* Definitions for GNU multiple precision functions.   -*- mode: c -*-
```

Comment:
Even though, that this header is something of an interface, we should consider adding an include guard, since this file is being included into a plethora of other headers. I know, that both `full-gmp.h` and `mini-gmp.h` contain include guards, but still, in my opinion, the lack of include guard in this file might be the cause of some really nasty bugs.